assisted-events-scrape/storage/cluster_events_storage.py

```diff
@@ -0,0 +1,158 @@
+import elasticsearch
+import os
+import re
+import time
+import hashlib
+import json
+from utils.logger import log
+import storage.process as process
+from events_scrape import InventoryClient
+
+MAX_EVENTS = 5000
+UUID_REGEX = r'[a-f0-9]{8}-?[a-f0-9]{4}-?4[a-f0-9]{3}-?[89ab][a-f0-9]{3}-?[a-f0-9]{12}'
+
+
+class ClusterEventsStorage:
+    @staticmethod
+    def create_with_inventory_client(inventory_client: InventoryClient, config: dict) -> 'ClusterEventsStorage':
+        elastic_config = config["elasticsearch"]
+        http_auth = None
+        if elastic_config["username"]:
+            http_auth = (elastic_config["username"], elastic_config["password"])
+        es_client = elasticsearch.Elasticsearch(elastic_config["host"], http_auth=http_auth)
+        return ClusterEventsStorage(
```

Comment:
for now it's ok, but the storage shouldn't contain the AI client

Reply:
I agree